milestone2.ts

```diff
@@ -92,23 +98,23 @@ let findAverage: GetNumber; // Define lambda function here
   Return -1 if no one is from San Francisco.
   Hint: Use a lambda function as a value
 */
-let findIndexAns: GetNumber; // Code here
+let findIndexAns: GetNumber = (data: Volunteer[]): number => data.findIndex(vol => vol.city === 'SF'); // findIndexAns function
```

Comment:
Here you are double-defining all of the types for this function. When you assigned the GetNumber type to findIndexAns it knows the types of all the inputs and outputs already, so you don't need to specify those again.